mk/kind.mk

```diff
@@ -116,6 +116,13 @@ kind/deploy/metrics: build/kumactl
 	@${BUILD_ARTIFACTS_DIR}/kumactl/kumactl install metrics $(KUMACTL_INSTALL_METRICS_IMAGES) | kubectl apply -f -
 	@KUBECONFIG=$(KIND_KUBECONFIG) kubectl wait --timeout=60s --for=condition=Ready -n kuma-metrics pods -l app=prometheus
 
+.PHONY: kind/deploy/metrics-server
+kind/deploy/metrics-server:
+	@KUBECONFIG=$(KIND_KUBECONFIG) kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.4.1/components.yaml
+	@KUBECONFIG=$(KIND_KUBECONFIG) kubectl patch -n kube-system deployment/metrics-server \
+		--patch='{"spec":{"template":{"spec":{"containers":[{"name":"metrics-server","args":["--cert-dir=/tmp", "--secure-port=4443", "--kubelet-insecure-tls", "--kubelet-preferred-address-types=InternalIP"]}]}}}}'
+	@KUBECONFIG=$(KIND_KUBECONFIG) kubectl wait --timeout=60s --for=condition=Available -n kube-system deployment/metrics-server
```

Comment:
So, how `kind` specific is this? You say in the readme `or on a cluster with a similarly self-signed certz`, what are these clusters? Is this a public cloud, or on-prem?
My point is, is, does this belong to `kind.mk` or can go a bit more generic?

Reply:
Definitely more generic, just wasn't sure which makefile to put it in -- maybe dev.mk? It should be up to the user to configure this separately from kuma, this make target is just meant to make the dev setup easier.

Reply:
This might also be the case for things like minikube and microk8s. Do you generally use kind for local dev?